packages/typespec-bump-deps/test/cli.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from "chai";
+import "mocha";
```

Comment:
Was this because typescript was complaining? I think you might need to add `mocha` to the tsconfig.json instead like https://github.com/microsoft/typespec/blob/4e685d2cd8141099d7337ec60b48a4fe6bb84454/packages/lint/tsconfig.json#L8

Reply:
I just copied the test file from https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-api/test/expectation.test.ts

Importing expect and mocha looks right, no?

Reply:
ah didn't realized that test was done this way, ideally you can just have it once in the tsconfig.json so each test doesn't need this import with side effects.

Or change it to be explicit like

```ts
import { describe, it } from "mocha";
```

Reply:
I think I prefer the explicit import